src/rt/dmain2.d

```diff
@@ -558,6 +558,31 @@ private void formatThrowable(Throwable t, scope void delegate(in char[] s) nothr
     }
 }
 
+private auto parseExceptionOptions()
+{
+    import rt.config : rt_configOption;
+    const optName = "trapExceptions";
+    auto res = rt_configOption(optName);
+    auto trap = rt_trapExceptions;
+    switch(res)
+    {
+        case "1", "y", "yes", "Y", "Yes", "true":
+            trap = true;
+            break;
+        case "0", "n", "no", "N", "No", "false":
+            trap = false;
+            break;
+        case "":
+            break;
+        default:
+            fprintf(stderr, "Invalid value '%.*s' for option '%s'.\n",
+                    res.length, res.ptr,
+                    optName.ptr);
+            break;
+    }
+    return trap;
+}
```

Comment:
Still quite verbose. How about exposing `parse` in core.internal.parseoptions via:
```
bool rt_parseOption(T...)(T args) { return parse(args); }
```
and then using
```
private auto parseExceptionOptions()
{
    import rt.config : rt_configOption;
    import core.internal.parseoptions : rt_parseOption;
    const optName = "trapExceptions";
    auto res = rt_configOption(optName);
    auto trap = rt_trapExceptions;
    if (res.length)
        rt_parseOption(optName, res, trap, "");
    return trap;
}
```